docs/input-data/crac/json.md

```diff
@@ -1187,23 +1190,27 @@ Note that the [PstHelper utility class](https://github.com/powsybl/powsybl-open-
 ~~~
 :::
 :::{group-tab} Object fields
-🔴⭐ **identifier**  
-⚪ **name**  
-⚪ **operator**  
-🔴 **network element**: id is mandatory, name is optional  
-⚪ **groupId**: if you want to align this range action with others, set the same groupId for all. You can use any
+- 🔴⭐ **identifier**  
+- ⚪ **name**  
+- ⚪ **operator**
+- ⚪ **activationCost**
+- ⚪ **variationCosts**
+- - ⚪ **up**
+- - ⚪ **down**
+- 🔴 **network element**: id is mandatory, name is optional  
+- ⚪ **groupId**: if you want to align this range action with others, set the same groupId for all. You can use any
 group ID you like, as long as you use the same for all the range actions you want to align.  
-🔵 **speed**: mandatory if it is an automaton  
-🔴 **initial tap**  
-🔴 **tap to angle conversion map**  
-🔴 **tap ranges**: list of 0 to N TapRange  
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 🔴 **range type**  
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 🔵 **min tap**  
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 🔵 **max tap**: at least one value must be defined  
-⚪ **onInstant usage rules**: list of 0 to N OnInstant usage rules (see paragraph on [usage rules](#remedial-actions-and-usages-rules))  
-⚪ **onState usage rules**: list of 0 to N OnContingencyState usage rules (see paragraph on [usage rules](#remedial-actions-and-usages-rules))  
-⚪ **onFlowConstraintInCountry usage rules**: list of 0 to N OnFlowConstraintInCountry usage rules (see previous paragraph on [usage rules](#remedial-actions-and-usages-rules))  
-⚪ **onConstraint usage rules**: list of 0 to N OnConstraint usage rules (see previous paragraph on [usage rules](#remedial-actions-and-usages-rules))  
+- 🔵 **speed**: mandatory if it is an automaton  
+- 🔴 **initial tap**  
+- 🔴 **tap to angle conversion map**  
+- 🔴 **tap ranges**: list of 0 to N TapRange  
+- - 🔴 **range type**  
+- - 🔵 **min tap**  
+- - 🔵 **max tap**: at least one value must be defined  
```

Comment:
doesn't look good
![image](https://github.com/user-attachments/assets/b0664c29-fd79-497a-857e-8e13bb3e13e0)

Reply:
nevermind this can be fixed by adding two white spaces at the end of each line